heroku/cli
# migrate(W-14170389): Upgrade  'pg:backups' index

This PR migrates `pg:backups` index to the CLI

## Test
1. checkout this branch and run `yarn && yarn build`
2. run `./bin/run pg:backups -a APP` choose an app with/without backups, with/without restores, with/without copies
3. Compare the output to the existing CLI.

Please note that this PR does not carry over a deprecated command alias structure that accepts sub-commands as arguments. This change was approved by the front-end team and the PG team.
